lesson_3/homework_03.py

```diff
@@ -0,0 +1,122 @@
+#Initial string
+#alice_in_wonderland = '"Would you tell me, please, which way I ought to go from here?"\n"That depends a good deal on where you want to get to," said the Cat.\n"I don't much care where ——" said Alice.\n"Then it doesn't matter which way you go," said the Cat.\n"—— so long as I get somewhere," Alice added as an explanation.\n"Oh, you're sure to do that," said the Cat, "if you only walk long enough."'
+
+# task 01 == Розділіть змінну alice_in_wonderland так, щоб вона займала декілька фізичних лінії
+alice_in_wonderland = """'"Would you tell me, please, which way I ought to go from here?"\n"That depends a good deal on where you want to get to," said the Cat.\n"I don't much care where ——" said Alice.\n"Then it doesn't matter which way you go," said the Cat.\n"—— so long as I get somewhere," Alice added as an explanation.\n"Oh, you're sure to do that," said the Cat, "if you only walk long enough."'"""
+
+# task 02 == Знайдіть та відобразіть всі символи одинарної лапки (') у тексті
+for character in alice_in_wonderland:
+    if character == "'":
+        print(character)
+
+# task 03 == Виведіть змінну alice_in_wonderland на
+print(alice_in_wonderland)
+
+
+"""
+    # Задачі 04 -10:
+    # Переведіть задачі з книги "Математика, 5 клас"
+    # на мову пітон і виведіть відповідь, так, щоб було
+    # зрозуміло дитині, що навчається в п'ятому класі
+"""
+# task 04
+"""
+Площа Чорного моря становить 436 402 км2, а площа Азовського
+моря становить 37 800 км2. Яку площу займають Чорне та Азов-
+ське моря разом?
+"""
+black_sea_area = 436402
+azov_sea_area = 37800
+print(f"Black and Azov seas cover {black_sea_area+azov_sea_area} km2")
```

Comment:
should be
```
print(f"Black and Azov seas cover {black_sea_area + azov_sea_area} km2")
```

Reply:
homework_3 file was updated based on feedback